zebra/redistribute.h

```diff
@@ -71,6 +71,9 @@ extern int is_zebra_import_table_enabled(afi_t, uint32_t table_id);
 
 extern int zebra_import_table_config(struct vty *);
 
-extern void zebra_import_table_rm_update(void);
+extern void zebra_import_table_rm_update(char *rmap);
 
+extern void zebra_rib_table_rm_update(char *rmap);
+
+extern void zebra_nht_rm_update(char *rmap);
```

Comment:
Why is this in redistribute.h?

Reply:
zebra_import_table_rm_update() is also in redistribute.h

Therefore added the new function prototypes in this file